src/components/AddressSearch.js

```diff
@@ -111,11 +138,28 @@ const AddressSearch = (props) => {
             }}
             textInputProps={{
                 InputComp: TextInput,
+                ref: (node) => {
+                    if (!props.innerRef) {
+                        return;
+                    }
+
```

Comment:
I think we can remove this check as `_.isFunction` would return false if `props.innerRef` is undefined.

Reply:
I think we should early return if there's no ref provided.

Just tried removing it and got this error:
<img width="646" alt="Screen Shot 2022-02-15 at 6 59 18 PM" src="https://user-images.githubusercontent.com/11542415/154076827-4920eb33-fc21-4b5c-a17c-bdfdeb846c50.png">